### PR TITLE
Update Pulumi Cloud branding

### PR DESCRIPTION
**Pull request by gthuang**
Update references to Pulumi Service to Pulumi Cloud

### CODE REVIEW

File: themes/default/content/registry/packages/pulumiservice/_index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Pulumi Service
+title: Pulumi Cloud (Pulumi Service provider)
```
Comment (susanev) [inline themes/default/content/registry/packages/pulumiservice/_index.md:2]: i dont think we should change this line until the package name changes

File: themes/default/data/registry/packages/pulumiservice.yaml [hunk 2]
```diff
@@ -9,6 +9,6 @@ package_status: public_preview
 publisher: Pulumi
 repo_url: https://github.com/pulumi/pulumi-pulumiservice
 schema_file_path: provider/cmd/pulumi-resource-pulumiservice/schema.json
-title: Pulumi Service
+title: Pulumi Cloud (Pulumi Service provider)
```
Comment (susanev) [inline themes/default/data/registry/packages/pulumiservice.yaml:12]: im not sure that we should change this, until the package name changes

**Review by susanev — COMMENTED** [review]
@gthuang im pretty worried about changing the name of this package before we actually change the name of both the package and the repo.

why do we need to make this change now?

File: themes/default/content/registry/packages/pulumiservice/_index.md (hunk 1 above)
Comment (gthuang) [reply themes/default/content/registry/packages/pulumiservice/_index.md:2]: that's why i kept both names in the title for discoverability purposes. when is the package being updated?

Comment (susanev) [reply themes/default/content/registry/packages/pulumiservice/_index.md:2]: @gthuang i dont know, youll have to work with @meagancojocar and @EvanBoyle on that

Comment (EvanBoyle) [reply themes/default/content/registry/packages/pulumiservice/_index.md:2]: I don't think there is much of a downside to this small change. We don't have a plan for doing the work to rename the provider yet.

Comment (susanev) [reply themes/default/content/registry/packages/pulumiservice/_index.md:2]: alright if yall dont think it will be confusing don't want to block.

Comment (gthuang) [reply themes/default/content/registry/packages/pulumiservice/_index.md:2]: thanks both, i will merge this PR

**Pull request merged**